ar4_isaac/launch/ar4_in_empty_world.launch.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# BSD 3-Clause License
+#
+# Copyright 2025 Ekumen, Inc.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+import os
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+
+
+def generate_launch_description():
+    pkg_andino_isaac_path = get_package_share_directory('ar4_isaac')
+    run_sim_path = os.path.join(pkg_andino_isaac_path, 'scripts', 'run_sim.py')
+
+    isaac_process = ExecuteProcess(
+        cmd=[
+            '/isaac-sim/python.sh',
+            run_sim_path,
+            '--robot_model_path',
+            os.path.join(pkg_andino_isaac_path, 'usda', 'ar4.usda'),
```

Comment:
Can we add a ros argument to be able to launch the simulation en headless/streaming mode?

That would allow poor fellows like me to run this in beefy/omni and access isaac using the streaming client.

Reply:
Done.

Reply:
For clarification, headless mode was added, I created [this issue](https://github.com/ekumenlabs/ar4/issues/58) for the streaming part.